src/components/Pagination/Pagination.js

```diff
@@ -82,11 +95,11 @@ export default class Pagination extends Component {
   handlePageInputChange = evt => {
     const page = Number(evt.target.value);
     if (
-      page > 0 &&
+      page >= 0 &&
```

Comment:
@joshblack Would you agree with the change here?

Reply:
OK is it possible for you to add what you described here to the code as a comment? Thanks!